### ISSUE TITLE
Redirect's prop 'from' not optional?

### ISSUE THREAD

**Issue by simonlc** [reach/router#26]
In the documentation, Redirect can "be rendered anywhere in your app when you want to declaratively redirect."

I usually get an error along the lines of proptypes falling because I am missing the `from` prop, or when I try to server render I get:

```
TypeError: Cannot read property 'replace' of undefined
    at stripSlashes
```

## Example

I made a quick example of how my code looks using your example if my comment isn't clear:

https://codesandbox.io/s/x75w229r5o

## Fix

I don't mind making a PR, though I had trouble understanding why the `from` prop is always required, so maybe I am missing something? That's why I didn't go right ahead and do it.

## Thanks

Thank you Ryan, another excellent library for React! The migration of my app took almost no time, I am just struggling with this last issue since I am not able to server render my pages.

**Comment by ryanflorence** [comment]
I'm sure I goofed up some code, `from` is required when rendered inside of `Router`, but not when rendered anywhere. PR would be great!

**Comment by ryanflorence** [comment]
fixed in v1.0.3
